internal/envoy/install.go

```diff
@@ -42,14 +41,14 @@ func InstallIfNeeded(o *globals.GlobalOpts, p, v string) (string, error) {
 			return "", fmt.Errorf("unable to create directory %q: %w", installPath, e)
 		}
 
-		m, e := manifest.GetManifest(o.ManifestURL)
+		ev, e := GetEnvoyVersions(o.EnvoyVersionsURL)
 		if e != nil {
 			return "", e
 		}
 
-		tarballURL, e := tarballURL(m, p, v)
-		if e != nil {
-			return "", e
+		tarballURL := ev.Versions[v].Tarballs[p]
```

Comment:
nit this would panic when `version, ok := ev.Versions[v]; !ok`.

Reply:
ah subtle but because this is no longer a graph of pointers, it "just works" missing tests though, so added them